scripts/add_ts_data_bidmatrixinfo.py

```diff
@@ -0,0 +1,84 @@
+###this script adds time series data to the bid matrix information objects in the powerops client
+
+from cognite.client.data_classes import TimeSeries
+from cognite.client.exceptions import CogniteNotFoundError
+from cognite.powerops import PowerOpsClient
+import random
+import datetime
+
+power = PowerOpsClient.from_settings()
+cdf = power.cdf
```

Comment:
**note:** these variable names aren't very descriptive, what is `power` or `cdf`?